src/search.ts

```
@@ -0,0 +1,137 @@
+     * @param results - An array of bang entries to display.
+     */
+    function displayResults(results: typeof bangs): void {
+        if (results.length === 0) {
+            resultsContainer.innerHTML = "No results found.";
+            return;
+        }
+        // Removed text-align from the ul to allow li items to span full width.
+        let listHtml = '<ul style="list-style:none;padding:0;margin:0;width:100%;">';
+        results.forEach(bang => {
+            listHtml += `<li style="padding:8px;border-bottom:1px solid #eee; text-align: center">` +
+                `<strong>${bang.s}</strong> - <em>!${bang.t}</em></li>`;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)